honcho/test/fixtures/custom_supervisord.conf

```diff
@@ -0,0 +1,22 @@
+# Generated from custom template: honcho/test/fixtures/custom_supervisord.conf
```

Comment:
I think having a simpler template just to test the `-t` functionality would be fine.